gdp-src-build/rokers-developer.conf

```diff
@@ -2,6 +2,11 @@
 
 BBLAYERS += " \
   ${TOPDIR}/../meta-rokers \
```

Comment:
Please keep the entries in `BBLAYERS` sorted alphabetically to make code maintenance easier.